.github/workflows/tests.yml

```diff
@@ -0,0 +1,34 @@
+name: Vitest, TypeScript check, ESLint
+
+on: [push, workflow_dispatch]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+        run:
+          working-directory: './client'
+  
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+            node-version: 23.x
```

Comment:
I would avoid using non-LTS versions in anything unless absolutely necessary.

Reply:
Ah it was the quickest fix for this... I'll look for a workaround
https://github.com/ietf-tools/rfced-www/actions/runs/12311686063/job/34362303820

Reply:
Done